src/components/MultiHopTrade/hooks/useReceiveAddress.tsx

```diff
@@ -67,6 +61,9 @@ export const useReceiveAddress = ({
       if (buyAssetChainId !== buyAssetAccountChainId) {
         return
       }
+
+      const fetchUnchainedAddress = Boolean(wallet && isLedger(wallet))
```

Comment:
This didnt have to be a prop - its always the same

Reply:
🥇